app/controllers/games_controller.rb

```diff
@@ -1,13 +1,25 @@
 # Main controller for chess session for CRUD
 # logic for our app
 class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :new]
+  def index
+  end
+
   def create
     @game = Game.new
   end
 
-  def index
+  def show
+    @game = Game.new
   end
 
   def new
+    @game = Game.new
+  end
```

Comment:
I don't foresee us using a new action but I think it's harmless to leave it for now.  We will probably have players create games straight from the index action (with either a 'Create Game' button or a modal form, depending on if and how we want to implement expanded actions the creator can take to customize the game.  I'd create a game object there instead.